subjects/java/checkpoints/factory-blueprint/README.md

```diff
@@ -0,0 +1,60 @@
+## Factory Blueprint
+
+### Instructions
+
+You are given an incomplete Factory design pattern implementation with some incorrect parts. Complete and fix the class to demonstrate your understanding of how the Factory design pattern works.
```

Comment:
There are missing details about the behaviour of `showDetails` for product A and product B. I see this won't be tested, but it could be nice to specify that to avoid confusions

Reply:
okay, no problem in that.